.github/workflows/pr-check.yml

```diff
@@ -8,35 +7,57 @@ on:
       - synchronize
       - closed
 
-jobs:
-  check:
-    runs-on: ubuntu-24.04
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
 
+jobs:
+  fmt:
+    runs-on: ubuntu-latest
+    container:
+      image: rust:1.87-slim
```

Comment:
Does that mean if we change the rusttoolchain file, we also need to change here?